From 998da4e12d08bb7d6727c9a8673dd21e98ddf5be Mon Sep 17 00:00:00 2001
Message-Id: <patch-2.2-998da4e12d0-20211119T124001Z-avarab@gmail.com>
In-Reply-To: <cover-0.2-00000000000-20211119T124001Z-avarab@gmail.com>
References: <cover-0.2-00000000000-20211119T124001Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 19 Nov 2021 09:17:00 +0100
Subject: [PATCH 2/2] CI: rename the "Linux32" job to lower-case "linux32"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

As a follow-up to the preceding commit's shortening of CI job names
rename the only job that starts with an upper-case letter to be
consistent with the rest. It was added in 88dedd5e72c (Travis: also
test on 32-bit Linux, 2017-03-05).

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 .github/workflows/main.yml        | 2 +-
 .travis.yml                       | 2 +-
 ci/install-docker-dependencies.sh | 2 +-
 ci/lib.sh                         | 2 +-
 ci/run-docker-build.sh            | 2 +-
 ci/run-docker.sh                  | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/.github/workflows/main.yml b/.github/workflows/main.yml
index 8f4caa8f040..9c4af979559 100644
--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -267,7 +267,7 @@ jobs:
         - jobname: linux-musl
           os: alpine
           image: alpine
-        - jobname: Linux32
+        - jobname: linux32
           os: ubuntu32
           image: daald/ubuntu32:xenial
         - jobname: pedantic
diff --git a/.travis.yml b/.travis.yml
index 908330a0a3d..73f983776c7 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -25,7 +25,7 @@ matrix:
       os: linux
       dist: trusty
       compiler:
-    - env: jobname=Linux32
+    - env: jobname=linux32
       os: linux
       compiler:
       addons:
diff --git a/ci/install-docker-dependencies.sh b/ci/install-docker-dependencies.sh
index 07a8c6b199d..78b7e326da6 100755
--- a/ci/install-docker-dependencies.sh
+++ b/ci/install-docker-dependencies.sh
@@ -4,7 +4,7 @@
 #
 
 case "$jobname" in
-Linux32)
+linux32)
 	linux32 --32bit i386 sh -c '
 		apt update >/dev/null &&
 		apt install -y build-essential libcurl4-openssl-dev \
diff --git a/ci/lib.sh b/ci/lib.sh
index 82cb17f8eea..83c2e08be79 100755
--- a/ci/lib.sh
+++ b/ci/lib.sh
@@ -222,7 +222,7 @@ osx-clang|osx-gcc)
 	;;
 linux-gcc-default)
 	;;
-Linux32)
+linux32)
 	CC=gcc
 	;;
 linux-musl)
diff --git a/ci/run-docker-build.sh b/ci/run-docker-build.sh
index 8d47a5fda3b..886fa33e148 100755
--- a/ci/run-docker-build.sh
+++ b/ci/run-docker-build.sh
@@ -15,7 +15,7 @@ then
 fi
 
 case "$jobname" in
-Linux32)
+linux32)
 	switch_cmd="linux32 --32bit i386"
 	;;
 linux-musl)
diff --git a/ci/run-docker.sh b/ci/run-docker.sh
index 37fa372052d..6663d56ade6 100755
--- a/ci/run-docker.sh
+++ b/ci/run-docker.sh
@@ -6,7 +6,7 @@
 . ${0%/*}/lib.sh
 
 case "$jobname" in
-Linux32)
+linux32)
 	CI_CONTAINER="daald/ubuntu32:xenial"
 	;;
 linux-musl)
-- 
2.34.0.817.gb03b3d32691

